app/controllers/show-project.js

```diff
@@ -53,12 +54,29 @@ export default class ShowProjectController extends Controller {
     } return false;
   }
 
-  // hearingsSubmitted is a helper that iterates through each disposition
-  // and checks whether disposition has dcpPublichearinglocation and dcpDateofpublichearing
-  @computed('model')
+  // if the first object dcpPublichearinglocation and dcpDateofpublichearing properties are filled,
+  // then hearings have been submitted for that project
+  @computed('model.dispositions.firstObject.{dcpPublichearinglocation,dcpDateofpublichearing}')
   get hearingsSubmittedForProject() {
-    const dispositions = this.get('model.dispositions');
-    return hearingsSubmitted(dispositions);
+    const location = this.get('model.dispositions.firstObject.dcpPublichearinglocation');
+    const date = this.get('model.dispositions.firstObject.dcpDateofpublichearing');
+    const locationAndDate = !!location && !!date;
+    return locationAndDate;
+  }
+
+  // if the first object dcpPublichearinglocation is "waived",
+  // then hearings have been waived
+  @computed('model.dispositions.firstObject.dcpPublichearinglocation')
+  get hearingsWaived() {
+    const firstObjectLocation = this.get('model.dispositions.firstObject.dcpPublichearinglocation');
+    const hearingsWaived = firstObjectLocation === 'waived';
+
+    return hearingsWaived;
```

Comment:
This is the sort of thing you can avoid declaring a new variable for and just say this:

`return firstObjectLocation === 'waived';`

I'm guess you were console logging the value of hearingsWaived, but that would best be done in a test, where you're testing the output value of it (controllers get unit tests for free).